src/main/java/page/url/PageUrl.java

```diff
@@ -0,0 +1,9 @@
+package page.url;
+
+public class PageUrl {
+    public static final String HOMEPAGE_URL = "https://magento.softwaretestingboard.com/";
+    public static final String CREATE_AN_ACCOUNT_URL = "https://magento.softwaretestingboard.com/customer/account/create/";
```

Comment:
There is already HOMEPAGE_URL,
so  CREATE_AN_ACCOUNT_URL should be  CREATE_AN_ACCOUNT_URL + customer/account/create/.
The same change should be done for other magento related urls in this class.